src/elements/tabs/tab-group/tab-group.ts

```diff
@@ -311,14 +326,15 @@ export class SbbTabGroupElement extends LitElement {
       tab.nextElementSibling?.localName &&
       SUPPORTED_CONTENT_WRAPPERS.includes(tab.nextElementSibling?.localName)
     ) {
-      tab.relatedContent = tab.nextElementSibling as HTMLElement;
+      tab.relatedContent = tab.nextElementSibling as SbbTabSupportedContentType;
       tab.relatedContent.id = this._assignId();
-      if (tab.relatedContent.nodeName !== 'SBB-TAB-GROUP') {
+      if (tab.relatedContent instanceof SbbTabElement) {
         tab.relatedContent.tabIndex = 0;
+        tab.relatedContent.configure();
       }
     } else {
-      tab.insertAdjacentHTML('afterend', '<div>No content.</div>');
-      tab.relatedContent = tab.nextElementSibling as HTMLElement;
+      tab.insertAdjacentHTML('afterend', '<sbb-tab>No content.</sbb-tab>');
```

Comment:
Not sure whether we still should do this. and I think, if yes, we should add the developer check to not have this in production build.

Reply:
moved some lines below together with the warning